packages/services/f-feature-management/test/endToEnd.test.js

```diff
@@ -0,0 +1,22 @@
+import featureManagement from '../src/index';
+import expectations from './data/e2e-expectations.json';
+
+const { getBooleanValue, getIntegerValue, getStringValue } = featureManagement('je-web-core');
+
+const functionMap = { boolean: getBooleanValue, integer: getIntegerValue, string: getStringValue };
+
+describe.skip('End-to-End Tests', () => {
+    describe.each(expectations)('Expectation %# - ', ({
+        description, expectation, feature, valueType
+    }) => {
+        it(description, () => {
+            const { value: expectedValue, experimentKey: expectedKey, experimentVariant: expectedVariant } = expectation;
+            const featureValue = functionMap[valueType](feature);
+            expect(featureValue.value).toBe(expectedValue);
+            if (expectedKey && expectedVariant) {
```

Comment:
I think we should try and avoid conditional expectations. Happy for you to update in your next PR though.

Reply:
I suppose I could change it to a falsy check of the value. I think in most cases the expected fields would be `null` whereas the `featureValue` properties would be `undefined`.

Yes, will mull it over and change for next PR 👍